sda-commons-server-morphia/src/test/java/org/sdase/commons/server/morphia/MorphiaBundleValidationIT.java

```diff
@@ -68,6 +69,7 @@ private Datastore getDatastore() {
 
     @Override
     public void initialize(Bootstrap<Config> bootstrap) {
+      bootstrap.addBundle(JacksonConfigurationBundle.builder().build());
```

Comment:
Do we know the reason why the Jackson bundle is now required everywhere? I guess it's not a big problem if everyone is using the starter bundle anyway.

Reply:
As @dhenneke mentioned yesterday: We probably haven't noticed it before because the new properties for metrics (e.g. 'duration') were only introduced recently.